domains/sendgrid.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "booic",
+        "email": "abd.21345206@commerce.menofia.edu.eg"
+    },
+    "record": {
+        "CNAME": "em2847.sendgrid.is-a.dev"
```

Comment:
you cant put a nested subdomain on the subdomain you are trying to get in the CNAME record, please change it to something else